src/modules/facility-fixed-fee/facility-fixed-fee.controller.ts

```diff
@@ -23,7 +35,49 @@ export class FacilityFixedFeeController {
   @ApiInternalServerErrorResponse({
     description: 'An internal server error has occurred.',
   })
-  async getFixedFeesForFacility(@Param() params: GetFacilityFixedFeeParamsDto): Promise<GetFacilityFixedFeeResponse> {
+  async getFixedFeesForFacility(@Param() params: FacilityFixedFeeParamsDto): Promise<GetFacilityFixedFeeResponse> {
     return await this.facilityFixedFeeService.getFixedFeesForFacility(params.facilityIdentifier);
   }
+
+  @Post('facilities/:facilityIdentifier/fixed-fees')
+  @ApiOperation({
+    summary: 'Create a new fixed fee for a facility.',
+  })
+  @ApiBody({
+    type: CreateFacilityFixedFeeRequestItem,
+    isArray: true,
+  })
+  @ApiCreatedResponse({
+    description: 'The fixed fee has been successfully created.',
+    type: CreateFacilityFixedFeeResponse,
+  })
+  @ApiNotFoundResponse({
+    description: 'The facility was not found.',
+  })
+  @ApiBadRequestResponse({
+    description: 'Bad request.',
+  })
+  @ApiInternalServerErrorResponse({
+    description: 'An internal server error has occurred.',
+  })
+  async createFixedFeeForFacility(
```

Comment:
Yes, ACBS checks if fixed fee exists and if it happens returns 400 `FixedFee exists`. I added tests for it and custom error message `Fixed fee with this period and lenderTypeCode combination already exist.` It can be found in todays commits.